docs/constitution.rst

```diff
@@ -0,0 +1,60 @@
+Mu's Constitution
+=================
+
+Our Values
+----------
+
+Happily, Mu's users and developers are a diverse and international community
+who maintain a reputation as a friendly, welcoming and dynamic group.
+
+We trust that such collaborators will treat each other in a way that reflects
+the widely help view that diversity and friendliness are strengths of our
+community to be celebrated and fostered.
+
+We believe anyone involved in Mu's community has a right to:
+
+* be treated with courtesy, dignity and respect;
+* be free from any form of discrimination, victimisation, harassment or
+  bullying.
+* enjoy an environment free from unwelcome behaviour, inappropriate language
+  and unsuitable imagery.
+
+To this end, we hope participants in our community abide by the spirit of our
+:doc:`code_of_conduct`.
+
+The Role of Administrators
+--------------------------
+
+Contributing to Mu
+------------------
+
```

Comment:
Strawman suggestion:

We're delighted for people to report problems, make suggestions and contribute code. Indeed we encourage it. We'd ask you to follow our development guidelines and, of course, to be respectful of our time.

It may be that we decide not to adopt some change you've proposed -- or even coded. Please don't take this personally. Any project, especially one with many users, has to measure contributions against its design ideals. And sometimes the two don't match.